### ISSUE TITLE
Poor Accuracy

### ISSUE THREAD

**Issue by deep0learning** [freeSubmission/MEDM#1]
Hi,

I run the code and I got very poor accuracy. Did I miss anything? Thanks in advanced.

learning rate： 0.0001

validation set: Average loss: 4.4136, Accuracy: 536/1200 (44.00%)

aeroplane: 93.0 bicycle: 66.0 bus: 91.0 car: 9.0 horse: 96.0 knife: 0.0 motorcycle: 90.0 person: 43.0 plant: 1.0 skateboard: 42.0 train: 4.0 truck: 1.0
average_acc 44.666666666666664
Experi-No: 1 source: train to target: validation max correct: 536 max true accuracy 44.00%  max average accuracy 44.67%
True Accs: [tensor(44)]
Avg Accs: [3.722222222222222]
True Max-Accs: [tensor(44)]
Avg Max-Accs: [3.722222222222222]
True Avg-Acc: tensor(44)
Avg-Acc: 3.722222222222222
class acc: [[93.0, 66.0, 91.0, 9.0, 96.0, 0.0, 90.0, 43.0, 1.0, 42.0, 4.0, 1.0]]
